tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -0,0 +1,172 @@
+"""
+Test to verify clone creation and deletion performance
+"""
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version
+)
+from ocs_ci.framework.testlib import (
+    performance, E2ETest
+)
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version('<4.6')
+@pytest.mark.parametrize(
+    argnames=["interface_type", "pvc_size", "file_size"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, '1', '600M'], marks=pytest.mark.polarion_id('OCS-2340')
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, '25', '15GB'], marks=pytest.mark.polarion_id('OCS-2340')
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, '50', '30GB'], marks=pytest.mark.polarion_id('OCS-2340')
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, '100', '60GB'], marks=pytest.mark.polarion_id('OCS-2340')
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, '1', '600M'], marks=pytest.mark.polarion_id('2341')
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, '25', '15GB'], marks=pytest.mark.polarion_id('2341')
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, '50', '30GB'], marks=pytest.mark.polarion_id('2341')
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, '100', '60GB'], marks=pytest.mark.polarion_id('2341')
+        )
+    ]
+)
+@performance
+class TestPVCSingleClonePerformance(E2ETest):
+    """
+    Test to verify single PVC clone creation and deletion performance
+    """
+
+    @pytest.fixture(autouse=True)
+    def base_setup(self, interface_type, pvc_size, pvc_factory, pod_factory):
+        """
+        create resources for the test
+        Args:
+            interface_type(str): The type of the interface
+                (e.g. CephBlockPool, CephFileSystem)
+            pvc_size: Size of the created PVC
+            pvc_factory: A fixture to create new pvc
```

Comment:
Adding fixtures as args doc string is not needed

Reply:
@AviadP - PLease elaborate. Do you mean that the dosctring for the fixture is not written properly? 
If you think that the docstring should be written in other way, please point me to the correct example ( in our tests).